### PR TITLE
Cv edit

### PR DESCRIPTION
**Pull request by Aclyoneus**
Till now there are sections up to experience part. The rest should be easier, because many things will be repeated in some way.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    display: flex;
+    justify-content: space-around;
+    font-size: 1.8rem;
+    font-weight: bold;
+}
+
+.navbar-link {
+    margin: 1rem 0;
+    text-decoration: none;
+    color: inherit;
```
Comment (mwanago) [inline style.css:60]: I think `color: black` would be more obvious here

File: style.css [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    box-sizing: border-box
+}
+
+.wrapper {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1200px;
+}
+
+a:visited {
```
Comment (mwanago) [inline style.css]: If you change it to
```
a:hover, a:visited, a:link, a:active {
  /* ... */
}
```
you will be able to simplify your `.navbar-link` styles

File: style.css [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    align-items: center;
+    flex-direction: column;
+    line-height: 2.5rem;
+}
+
+.personals-text__line {
+    font-size: 1.8rem;
+}
+
+span {
```
Comment (mwanago) [inline style.css]: Don't write CSS that generic. If you forget about it later and add a `<span>` element, this CSS style will apply to it

File: index.html [hunk 4]
```diff
@@ -35,9 +40,79 @@ <h2 class="title">
...
+                            <img class="qrcode-image" src="img/linkedin.png">
+                            <a class="qrcode-description" href="https://www.linkedin.com/in/wojciech-wr%C3%B3bel-034085166" target="_blank">Linkedin</a>
+                        </div>
+                        <div class="personals-text">
+                            <h2 class="personals-text__name">wojciech wróbel</h2>
+                            <div class="personals-text__data">
+                                <p class="personals-text__line birth-date"><span>Date of Birth:</span> xx.xx.xxxx</p>
+                                <p class="personals-text__line phone"><span>Phone:</span> xxx-xxx-xxx</p>
+                                <p class="personals-text__line mail"><span>E-mail:</span> xxxxxxxx@gmail.com</p>
+                                <p class="personals-text__line residence"><span>Place of Residence:</span> Warsaw</p>
```
Comment (mwanago) [inline index.html]: Don't rely on `display: inline` from `<span>`. Use Flexbox instead if necessary

File: style.css [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+}
+
+.experience-company {
+    margin-top: 2rem;
+    margin-bottom: 0.8rem;
+    font-size: 2rem;
+}
+
+.experience-position {
+    display: grid;
```
Comment (mwanago) [inline style.css]: Don't use `grid` yet - practice your Flexbox skills. You can surely achieve the same thing with Flexbox

File: style.css [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+    list-style-position: inside;
+    margin-top: 0.8rem;
+    padding-left: 2rem;
+}
+
+.fa-solid {
+    transition: color 0.3s;
+}
+
+.experience-card:hover .fa-solid{
```
Comment (mwanago) [inline style.css]: There is a missing space in this line

File: style.css (hunk 2 above)
Comment (Aclyoneus) [reply style.css]: Done

File: style.css [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+
+/*SKILLS*/
+
+/*HOBBIES*/
+
+/*FOOTER*/
+
+.footer-permission {
+    padding: 1.5rem 0;
+    background-color: ;
```
Comment (mwanago) [inline style.css]: There seems to be something wrong with this line

**Review by mwanago — APPROVED** [review]

**Pull request merged**